biz.aQute.bndlib/src/aQute/bnd/osgi/Analyzer.java

```diff
@@ -1810,10 +1811,9 @@ protected void begin() {
 		if (inited == false) {
 			inited = true;
 			super.begin();
-
 			updateModified(getBndLastModified(), "bnd last modified");
+			About.fixup(this);
```

Comment:
Should it really be part of the Analyzer and not the JPMSPlugin fetching its default value from the About.getDefaults()?

Reply:
No, otherwise this fails for everything

Reply:
Then calling this once in `aQute.bnd.osgi.Processor.getProperties0()` might be more appropriate...